spec/RegonSpecs.js

```diff
@@ -0,0 +1,36 @@
+describe("Regon tests", function () {
+    var generateDigitRegon = require('../regon.js');
+
```

Comment:
General comment: you have several tests but non of them checks if regon with control digit is really returned by the function. You check length and last digit but never complete regon